packages/camera/camera/ios/Classes/CameraPlugin.h

```diff
@@ -5,4 +5,5 @@
 #import <Flutter/Flutter.h>
 
 @interface CameraPlugin : NSObject <FlutterPlugin>
+@property(nonatomic, strong, nullable) dispatch_queue_t dispatchQueue;
```

Comment:
Make this readonly and comment it is for testing only.

Reply:
actually, do this instead : https://github.com/flutter/plugins/pull/4619#discussion_r770928353